BenadrylAutomation/testng.xml

```diff
@@ -4,9 +4,11 @@
 	<listeners>
 		<listener class-name="com.jnj.utility.CustomListener" />
 	</listeners>
+
  	<test name="HEADER PAGE">
 		<classes>
 			<class name="com.jnj.testcases.HeaderPageTest" />
+			<class name="com.jnj.testcases.SignUpPageTest" />
```

Comment:
@vaibhavNagvekarSJI please revert the testng format to old one

Reply:
@goutamnaik007 fixed

Reply:
@vaibhavNagvekarSJI verified